atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -368,6 +368,12 @@ void WebFrame::ClearCache(v8::Isolate* isolate) {
     base::MemoryPressureListener::MEMORY_PRESSURE_LEVEL_CRITICAL);
 }
 
+void WebFrame::SetCacheCapacity(size_t capacity) {
+  if (capacity > 0) {
```

Comment:
Can a caller set size to 0?

Reply:
(with my lacks of knowledge around codebase) it seems size_t bind allows conversion from 0. I couldn't see specific inside of setcapacity blows up with 0, preventively added guard - but with comment I added above, thinking maybe it'd better to throw instead of guarding values.